controller/src/main/java/org/jboss/as/controller/remote/ModelControllerClientOperationHandler.java

```diff
@@ -68,7 +69,12 @@
  */
 public class ModelControllerClientOperationHandler implements ManagementRequestHandlerFactory {
 
-    private static final Set<String> PREPARED_RESPONSE_OPERATIONS = Set.of(RELOAD, SHUTDOWN);
+    private static final Set<String> PREPARED_RESPONSE_OPERATIONS;
+    static {
+        Set<String> ops = new HashSet<>(RELOAD_OPERATIONS);
+        ops.add(SHUTDOWN);
+        PREPARED_RESPONSE_OPERATIONS = Collections.unmodifiableSet(ops);
```

Comment:
This is very minor and in this specific case likely not a big deal, but you're going to see less of a memory footprint with `PREPARED_RESPONSE_OPERATIONS = Set.of(SHUTDOWN)`. The performance may be better too.

Reply:
Sorry, I completely missed the argument on the constructor. This is completely fine, ignore me :)

Reply:
It is a static initialiser called once, and the code adds an additional entry (stored in RELOAD_OPERATIONS to not have to duplicate everywhere) rather than remove one :-)